site/content/docs/5.1/components/carousel.md

```diff
@@ -152,7 +152,7 @@ Add captions to your slides easily with the `.carousel-caption` element within a
 
 ### Crossfade
 
-Add `.carousel-fade` to your carousel to animate slides with a fade transition instead of a slide.
+Add `.carousel-fade` to your carousel to animate slides with a fade transition instead of a slide. Depending on your carousel content (e.g., text only slides), you may way to add `.bg-body` or some custom CSS to the `.carousel-item`s for proper crossfading.
```

Comment:
```suggestion
Add `.carousel-fade` to your carousel to animate slides with a fade transition instead of a slide. Depending on your carousel content (e.g., text only slides), you may want to add `.bg-body` or some custom CSS to the `.carousel-item`s for proper crossfading.
```